c2corg_ui/templates/helpers/view.html

```diff
@@ -453,12 +453,12 @@ <h3 class="heading show-phone outings" ng-click="mainCtrl.animateHeaderIcon($eve
   % endif
 </%def>
 
-<%def name="get_document_description(locale, doctype)">\
+<%def name="get_document_description(locale, doctype, title=None)">\
   % if locale.get('summary') or locale.get('description'):
     <div class="view-details-description col-xs-12  description">
       % if doctype not in ['article', 'profile']:
         <h3 class="heading show-phone">
-          <span translate>description</span>
+          <span x-translate>${title if title else 'description'}</span>
```

Comment:
OK because ``description`` is probably extracted somewhere else (?) but a "safer" syntax would have been:
```
% if title:
  <span x-translate>${title}</span>
% else:
  <span translate>description</span>
% endif
```